docs/learn/learn-auction.md

```diff
@@ -31,16 +31,37 @@ extinguished and the candle went out, the auction would suddenly terminate and t
 that point would win.
 
 When candle auctions are used online, they require a random number to decide the moment of
-termination.
+termination. Parachain slot auctions differ slightly from a normal candle auction in that it does not 
+use the random number to decide the duration of its opening phase. 
+
+The candle auction on Polkadot is split into two parts: the *opening period* which is in 
+effect immediately after the auction starts. This period lasts for one day and eighteen hours and 
+serves as a buffer time for parachain candidates to setup their initial bids, and likely start executing 
+their strategy on how to win the slot auction. During the opening phase, bids will continue to be accepted. 
+During the ending period, later bids have a higher probability of losing since they are retroactively 
```

Comment:
I'd suggest removal of this sentence altogether. The next paragraph formally introduces the ending period. Also, the sentence _later bids have high probability of losing_ is a bit vague. The probability of winning the auction is equal to the **number of blocks with a winning bid** divided by the **total number of blocks in the ending period**.

Reply:
I still see this factually misleading statement that might need to be rephrased - - "During the ending period, 
later bids have a higher probability of losing since they are retroactively determined close moment may 
be found to have preceded the time that a bid was submitted."

A statement like this would remove the ambiguity -
"The probability of winning the auction is equal to the number of blocks with a winning bid divided by the total number of blocks in the ending period."